docs/internal/contributing/README.md

```diff
@@ -91,7 +99,7 @@ Please see the dedicated "[Design patterns and Code conventions](design-patterns
 
 ## Documentation
 
-The Grafana Mimir documentation is compiled into a website published at [grafana.com](https://grafana.com/). Run `make docs` to build and serve the documentation locally.
+The Grafana Mimir documentation is compiled into a website published at [grafana.com](https://grafana.com/). Run `make docs` to build and serve the documentation locally. For more detail on style and organisation of the documentation, refer to the dedicated page "[How to write documenentation](how-to-write-documentation.md)".
```

Comment:
Can we use Mimir-specific address, and spell it out fully like this: [https://grafana.com/docs/mimir/latest/](https://grafana.com/docs/mimir/latest/). It adds burden to us to keep it updated, but it also tells people where to find it.

Reply:
I could be missing something, but I don't think that file is part of the compiled documentation at all? Not sure if that is intended.

Reply:
That link points to webpage generated from `docs/sources/mimir/_index.md` file in this repository. Perhaps I misunderstand what you mean?

Reply:
Ok, I got it now :) fixed in https://github.com/grafana/mimir/pull/4079/commits/f37e241606750f1250b676749f3853846b264d7b.